controllers/openstackdataplanedeployment_controller.go

```diff
@@ -102,9 +101,6 @@ func (r *OpenStackDataPlaneDeploymentReconciler) Reconcile(ctx context.Context,
 		r.Scheme,
 		logger,
 	)
-	if err != nil {
```

Comment:
Either this, or we should replace the null identifier with err when we initialise the helper.